front/src/components/ArtworkCard.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="container">
+    <router-link :to="/artwork/ + id">
+      <img class="image" :src="image" alt="IMAGE" />
+      <div class="overlay">
+        <h3 class="title">{{ title }}</h3>
+        <p class="location">{{ location }}</p>
+      </div>
+    </router-link>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ArtworkCard",
+  props: {
+    id: {
+      type: Number,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    location: {
+      type: String,
+      required: true,
+    },
+    image: {
+      type: String,
+      required: true,
+    },
+  },
```

Comment:
It would be better if this component would just take a full Artwork object (json), please replace the props by : 
```js
props: {
   artwork: {
     type: Object,
     required: true
   }
}
```
Adjust if necessary.